kuma/static/js/utils/utils.js

```diff
@@ -0,0 +1,21 @@
+window.mdn.utils = {
+    /**
+     * Generate and returns a random string thanks to:
+     * https://stackoverflow.com/questions/1349404/generate-random-string-characters-in-javascript
+     * @param {Number} strLength - The length of the string to return
+     * @return {String} a randomly generated string with a chracter count of `strLength`
+     */
+    randomString: function(strLength) {
```

Comment:
This refactored code works in the console. ``randomString()`` now returns an empty string, but all the calls have been updated to ``mdn.utils.randomString(5)``.  I have a slight preference for a default, like ``strLength = Math.min(5, strLength)``.

Reply:
Yeah, I agree. My initial thought was to provide a default if nothing is passed, I should do that.